plugin/adsk/plugin/plugin.cpp

```diff
@@ -78,7 +85,7 @@ MAYAUSD_PLUGIN_PUBLIC
 MStatus initializePlugin(MObject obj)
 {
     MStatus status;
-    MFnPlugin plugin(obj, "Autodesk", "1.0", "Any");
+    MFnPlugin plugin(obj, "Autodesk", TOSTRING(MAYAUSD_VERSION), "Any");
```

Comment:
This will be replaced with `"0.3.0"`

Reply:
@seando-adsk  Nice, makes sense. Thanks.